### PR TITLE
Guidance for support squad: where to leave docs feedback

### PR DESCRIPTION
**Pull request by sarah11918**
This PR adds a section to the Support Squad guide describing how to give docs feedback they collect while helping people in support threads

### CODE REVIEW

**Review by TheOtterlord — APPROVED** [review]

**Review by BryceRussell — APPROVED** [review]
LGTM!

**Review by HiDeoo — APPROVED** [review]
Great additions 🌟

File: src/content/docs/roles/support-squad.md [hunk 1]
```diff
@@ -28,13 +28,20 @@ Be very clear that you are not giving off RTFM vibes! You *can* give a short, ye
...
   - If no, does this page have what you need(ed)? If you had seen this in docs first, would you have still needed to ask a support question? (And if so, would it have been a different question?)
 - Is there anything that docs could have provided or done differently so that you wouldn't have needed to come ask here? (e.g. have a section / example about this; have better navigation to this location; have this page ranked higher in Algolia search results)
 - "Would you be wiling to leave feedback about this page using the Feedback widget, or would you like me to leave feedback on your behalf based on our conversation?" (Note: the feedback widget allows someone to choose leaving feedback directly in the widget OR filing a GitHub issue, so sending them there works for either situation.)
 
+### Reporting Feedback to Docs
+
+If your support interaction reveals that docs information is incorrect, outdated, or confusing then we consider this an issue to be fixed. File a GitHub issue directly, or through the feedback widget on the docs site.
```
Comment (sarah11918) [inline src/content/docs/roles/support-squad.md]:
```suggestion
If your support interaction reveals that docs information is incorrect, outdated, or confusing then we consider this an issue to be fixed. [File a GitHub issue directly](https://github.com/withastro/docs/issues/new/choose), or through the feedback widget on the docs site.
```

**Comment by zougari47** [comment]
LGTM

**Pull request merged**